docker/object-detection-tf-py.debian-9/Dockerfile

```diff
@@ -0,0 +1,103 @@
+    LANG=C.UTF-8
+RUN mkdir -p ${CK_ROOT} ${CK_REPOS} ${CK_TOOLS}
+RUN git config --global user.name ${GIT_USER} && git config --global user.email ${GIT_EMAIL}
+RUN git clone https://github.com/ctuning/ck.git ${CK_ROOT}
+RUN cd ${CK_ROOT} \
+ && ${CK_PYTHON} setup.py install --user \
+ && ${CK_PYTHON} -c "import ck.kernel as ck; print ('Collective Knowledge v%s' % ck.__version__)"
+
+# Pull CK repositories (including ck-env, ck-autotuning and ck-tensorflow).
+RUN ck pull repo:ck-mlperf
+
+# Use generic Linux settings with dummy frequency setting scripts.
+RUN ck detect platform.os --platform_init_uoa=generic-linux-dummy
+
+# Detect C/C++ compiler (gcc).
+RUN ck detect soft:compiler.gcc --full_path=`which ${CK_CC}`
+
+# Detect Python.
+RUN ck detect soft:compiler.python --full_path=`which ${CK_PYTHON}`
+# Install the latest Python package installer (pip) and some dependencies.
+RUN ${CK_PYTHON} -m pip install --ignore-installed pip setuptools --user
+
+# Install Python dependencies.
+RUN ck install package --tags=lib,python-package,numpy
+RUN ck install package --tags=lib,python-package,scipy --force_version=1.2.1
+RUN ck install package --tags=lib,python-package,pillow
+RUN ck install package --tags=lib,python-package,matplotlib
+RUN ck install package --tags=lib,python-package,cython
+RUN ck show env --tags=python-package
+
+RUN ${CK_PYTHON} -m pip install gast --user
+RUN ${CK_PYTHON} -m pip install astor --user
+RUN ${CK_PYTHON} -m pip install termcolor --user
+RUN ${CK_PYTHON} -m pip install tensorflow-estimator==1.13.0 --user
+RUN ${CK_PYTHON} -m pip install keras_applications==1.0.4 --no-deps --user
+RUN ${CK_PYTHON} -m pip install keras_preprocessing==1.0.2 --no-deps --user
+# Install Python COCO API.
+RUN ck install package --tags=tool,coco,api
+
+# Install TF.
+RUN ck install package:lib-tensorflow-1.13.1-cpu
+# Download the SSD-MobileNet TF models (non-quantized and quantized).
+# https://github.com/mlperf/inference/blob/master/edge/object_detection/ssd_mobilenet/tflite/README.md#install-the-ssd-mobilenet-models-for-tflite
+RUN ck install package --tags=model,tf,object-detection,mlperf,ssd-mobilenet,non-quantized
+#RUN ck install package --tags=model,tf,object-detection,mlperf,ssd-mobilenet,non-quantized
+#RUN ck install package --tags=model,tf,object-detection,mlperf,ssd-mobilenet,quantized,finetuned
+
+# Download the COCO 2017 validation dataset (5,000 images) to the default path;
+# preprocess all the 5000 images (as the user may want to run full evaluation);
+# remove the original images and training annotations while taking care to preserve
+# 'val2017/000000000139.jpg' used to detect COCO 2017 (ck-env:soft:dataset.coco.2017.val).
+RUN echo | ck install package --tags=object-detection,dataset,coco.2017,val,original,full
+# && ck install package --tags=object-detection,dataset,coco.2017,preprocessed,first.50 \
+# && ck virtual env --tags=object-detection,dataset,coco.2017,val,original,full --shell_cmd=\
```

Comment:
You can leave this command but do the last thing only, namely, remove the training annotations:
```
$ rm $CK_ENV_DATASET_COCO_LABELS_DIR/*train2017.json'
```